src/UI/atoms/Text.tsx

```diff
@@ -12,11 +12,14 @@ import { ShuffleState } from "<hooks>/__typings__/useShuffleText.d.ts";
 function Text({
   children,
   color = "blue300",
+  dataTestId,
   ellipsis = false,
   fontFamily = "AnonymousPro",
   fontSize = "font20",
   fontWeight = "normal",
   lineHeight = "1",
+  maxHeight,
+  overflow,
```

Comment:
```suggestion
  overflow = "visible",
```
There should be a default value